docs/content/modeling/parent-child.mdx

```diff
@@ -256,9 +260,12 @@ The chain of resolution becomes:
 - Therefore, **bob** is an `editor` of **document:meeting_notes.doc**
 
 :::caution
-**Note:** When creating relationship tuples for <ProductName format={ProductNameFormat.LongForm}/> make sure to use unique ids for each object and user within your application domain. We are using first names and simple ids to just illustrate an easy-to-follow example.
+Note that <ProductName format={ProductNameFormat.LongForm}/> does not evaluate the related object (the word after `from`, also called the tupleset), it will only consider directly related objects, and will not perform any evaluation (e.g. any rewrites on that relation, or `*`, etc..).
```

Comment:
See discussion in configuration-language.mdx, directly related object may be confused with direct relationship.